File: Compendus/Packages/whisper.cpp/.build/index-build/arm64-apple-macosx/debug/whisper.build/DerivedSources/resource_bundle_accessor.m
#import <Foundation/Foundation.h>

NSBundle* whisper_SWIFTPM_MODULE_BUNDLE() {
    NSURL *bundleURL = [[[NSBundle mainBundle] bundleURL] URLByAppendingPathComponent:@"whisper_whisper.bundle"];

    NSBundle *preferredBundle = [NSBundle bundleWithURL:bundleURL];
    if (preferredBundle == nil) {
      return [NSBundle bundleWithPath:@"/Users/gabrielcsapo/Documents/coding/Compendus/Compendus/Packages/whisper.cpp/.build/index-build/arm64-apple-macosx/debug/whisper_whisper.bundle"];
    }

    return preferredBundle;
}

File: Compendus/EPUBReader/.build/arm64-apple-macosx/debug/EPUBReader.build/DerivedSources/resource_bundle_accessor.swift
import Foundation

extension Foundation.Bundle {
    static let module: Bundle = {
        let mainPath = Bundle.main.bundleURL.appendingPathComponent("EPUBReader_EPUBReader.bundle").path
        let buildPath = "/Users/gabrielcsapo/Documents/coding/Compendus/Compendus/EPUBReader/.build/arm64-apple-macosx/debug/EPUBReader_EPUBReader.bundle"

        let preferredBundle = Bundle(path: mainPath)

        guard let bundle = preferredBundle ?? Bundle(path: buildPath) else {
            // Users can write a function called fatalError themselves, we should be resilient against that.
            Swift.fatalError("could not load resource bundle: from \(mainPath) or \(buildPath)")
        }

        return bundle
    }()
}